pkg/controller/controller.go

```diff
@@ -18,4 +18,9 @@ type Args struct {
 	// RevisionLimit is the maximum number of revisions that will be maintained.
 	// The default value is 50.
 	RevisionLimit int
+
+	// LimitedReconcile indicates whether reconcile periodically or only reconcile
+	// when specific fields are changed, e.g., metadata.labels, metadata.annotations,
+	// and spec.
+	LimitedReconcile bool
```

Comment:
I suggest we name it `ReconcileByChangeOnly`

Reply:
Got it.